src/routes/AppRouter.tsx

```diff
@@ -20,6 +22,16 @@ const router = createBrowserRouter([
         element: <Redirect to="wallet/home" />,
       },
 
+      {
+        path: 'success',
+        Component: Success,
```

Comment:
'success' is very generic. is it related to 'payment'? might be it should be renamed to something like 'PaymentSuccess' or 'PaymentCancel'?